### PR TITLE
Facts over falsehood

### PR DESCRIPTION
**Pull request by cwilso**
partially addresses #113

This is an attempt to avoid the inference of censorship.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/AB-public/pull/125.html" title="Last updated on Sep 12, 2023, 4:33 PM UTC (bc68970)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/AB-public/125/b3380a1...bc68970.html" title="Last updated on Sep 12, 2023, 4:33 PM UTC (bc68970)">Diff</a>

### CODE REVIEW

File: Vision/Vision.bs [hunk 1]
```diff
@@ -86,7 +86,7 @@ Boilerplate: omit conformance
 	as it expands reach, knowledge, education, and services even more broadly.
 	We believe the World Wide Web should be 
 	inclusive and respectful of its users: 
-	a Web that supports truth over falsehood, 
+	a Web that supports facts over falsehood,
```
Comment (dwsinger) [inline Vision/Vision.bs:89]: minor, they should be both plural or both singular, and it's punchier as "fact over falsehood". Perhaps that it's a direction of optimization would be clearer if it said "promotes" or "prefers" or "values"

**Review by tantek — CHANGES_REQUESTED** [review]
Note that the subsequent clause is also plurals: "people over profits,"

File: Vision/Vision.bs (hunk 1 above)
Comment (cwilso) [reply Vision/Vision.bs]: made both plural.

**Comment by dontcallmedom** [comment]
it's an improvement, but I don't think it addresses the core of the issue I perceive; I'll follow up on #113

**Pull request merged**